core/src/main/java/io/fundrequest/core/request/fund/domain/PendingFund.java

```diff
@@ -42,6 +45,11 @@
     @Embedded
     private IssueInformation issueInformation;
 
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "added_date")
+    @Builder.Default
+    private Date addedDate = new Date();
```

Comment:
extend from AbstractEntity, this will add meta info like creation_date which can be used